docs/rum_collection/advanced_configuration.md

```diff
@@ -0,0 +1,370 @@
+---
+title: RUM iOS Advanced Configuration
+kind: documentation
+further_reading:
+  - link: "https://github.com/DataDog/dd-sdk-ios"
+    tag: "Github"
+    text: "dd-sdk-ios Source code"
+  - link: "/real_user_monitoring"
+    tag: "Homepage"
+    text: "Explore Datadog RUM"
+---
+
```

Comment:
For Browser, we change the name of this page to Modifying Data and Context, which is more meaningful that "Advanced Configuration." Maybe we could do something like that here, too?

Reply:
Here I'm not sure 💭. We have a bunch of methods for **manual** instrumentation of views, actions, errors and resources which are described here. Also our **auto** instrumentation is highly customizable, which we also explain in on this page along with other configuration parameters for the SDK. It's very handy to have and "Advanced Configuration" page for existing and upcoming APIs.

When it comes to actual "modifying data" this is indeed sth that we may want to move to separate page. I will think of this and propose an update 👍.

Reply:
As first step, we should keep "Advanced configuration" (not the best naming, but better than manual configuration), then as step 2 we can plan to break sections similar to browser into - track errors, track resources including both auto and advanced config details...